Doc/library/enum.rst

```diff
@@ -267,6 +267,10 @@ Data Types
          appropriate value will be chosen for you.  See :class:`auto` for the
          details.
 
+         If the value is not hashable, the enum creation will take quadratic time
+         relative to the number of members.
```

Comment:
I think I might express this as something like:

> ... the time required to enumerate the members will grow quadratically with the number of members.

Reply:
Updated wording to include this as a warning.